src/main/java/org/wildfly/security/http/util/sso/ImmutableSingleSignOn.java

```diff
@@ -53,8 +52,8 @@
     SecurityIdentity getIdentity();
 
     /**
-     * Returns the participants associated with this single sign-on entry
+     * Returns the participants associated with this single sign-on entry.
      * @return an unmodifiable mapping of application identifier to a tuple of the session identifier and request URI
      */
-    Collection<Map.Entry<String, URI>> getParticipants();
+    Map<String, Map.Entry<String, URI>> getParticipants();
```

Comment:
N.B. This API change will affect the DistributableSingleSignOn implementation in WF.  I'll prepare a PR for that.